src/window/Window.cpp

```diff
@@ -286,21 +69,11 @@ std::shared_ptr<std::vector<WindowBackend>> Window::GetAvailableWindowBackends()
 
 void Window::SetWindowBackend(WindowBackend backend) {
     mWindowBackend = backend;
-    Context::GetInstance()->GetConfig()->SetWindowBackend(GetWindowBackend());
-    Context::GetInstance()->GetConfig()->Save();
+    Ship::Context::GetInstance()->GetConfig()->SetWindowBackend(GetWindowBackend());
+    Ship::Context::GetInstance()->GetConfig()->Save();
```

Comment:
not complaining, but we don't actually need `Ship::` here because `Window` is in the ship namespace right? (seeing this just made me do a double take and verify we were actually in the ship namespace here)